From f0224e42008535564f0d59ad02e24be3fc8cbf39 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Wed, 10 Aug 2022 01:57:24 +0200
Subject: [PATCH 4/5] Use krb5_auth_con_getsendsubkey() instead of
 krb5_auth_con_getlocalsubkey()

The latter is not exposed in the headers anymore.

Origin: vendor, Debian
Forwarded: no
---
 libinetutils/kerberos5.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libinetutils/kerberos5.c b/libinetutils/kerberos5.c
index 5df12e77..f77bc412 100644
--- a/libinetutils/kerberos5.c
+++ b/libinetutils/kerberos5.c
@@ -154,7 +154,7 @@ kerberos_auth (krb5_context *ctx, int verbose, char **cname,
   krb5_data_free (&cksum_data);
 # endif
 
-  rc = krb5_auth_con_getlocalsubkey (*ctx, auth_ctx, key);
+  rc = krb5_auth_con_getsendsubkey (*ctx, auth_ctx, key);
 
   /* send size of AP-REQ to the server */
 
-- 
2.43.0

